ptf_nn/ptf_nn_agent.py

```diff
@@ -163,10 +189,40 @@ def get_mac(self):
     def get_ctrs(self):
         return self.rx_ctr, self.tx_ctr
 
+    def port_up(self):
+        set_if_status(self.iface_name, True)
+        logger.debug("IfaceMgr {}-{} ({}) status set to UP".format(
+                     self.dev, self.port, self.iface_name))
+
+    def port_down(self):
+        set_if_status(self.iface_name, False)
+        logger.debug("IfaceMgr {}-{} ({}) status set to DOWN".format(
+                     self.dev, self.port, self.iface_name))
+
     def run(self):
         while True:
```

Comment:
I'm confused by why this outer `while` is needed? Could you add a comment to explain?